### ISSUE TITLE
Errors polling for event status: failed to query event: CUDA_ERROR_DEINITIALIZED

### ISSUE THREAD

**Issue by jw447** [tensorflow/tensorflow#8483]
I am running Tensor-flow on Google compute engine instances and I met with this problem when the training period is finished (after step 9990).
```
2017-03-17 00:24:23.021298: step 9980, loss = 0.72 (242.4 examples/sec; 0.264 sec/batch)
2017-03-17 00:24:34.693612: step 9990, loss = 0.79 (208.4 examples/sec; 0.307 sec/batch)
E tensorflow/stream_executor/cuda/cuda_event.cc:49] Error polling for event status: failed to query event: CUDA_ERROR_DEINITIALIZED
F tensorflow/core/common_runtime/gpu/gpu_event_mgr.cc:198] Unexpected Event status: 1
Aborted (core dumped)
```
Can anyone identify this problem and let me know how to fix it?

**Comment by poxvoculi** [comment]
Marking as duplicate of #8517
